MerlinAU.sh

```diff
@@ -2237,73 +2270,120 @@ _Set_FW_UpdatePostponementDays_()
    return 0
 }
 
+##-------------------------------------##
+## Added by Martinski W. [2024-Feb-22] ##
+##-------------------------------------##
+_ValidateCronJobSchedule_()
+{
+   local cronSchedsStr
+   if [ $# -eq 0 ] || [ -z "$1" ]
+   then
+       printf "${REDct}INVALID cron schedule string: [EMPTY].${NOct}\n"
+       return 1
+   fi
+   cronSchedsStr="$(echo "$1" | awk -F ' ' '{print NF}')"
+   if [ "$cronSchedsStr" -ne 5 ]
+   then
+       printf "${REDct}INVALID cron schedule string [$1]. Incorrect number of parameters.${NOct}\n"
+       return 1
+   fi
+   cronSchedsStr="$(echo "$1" | awk -F ' ' '{print $1}')"
```

Comment:
exhaustive validation alright lol!

Reply:
> exhaustive validation alright lol!
>
Well, it's not quite an exhaustive check & validation; it's more like a syntax parser validator so it will flag the most common user errors WRT syntax, but a user can still set a "syntactically correct" schedule that it will not work like:

10  3  31  2  0
10  3  31  Feb  Sun

The above entries are obviously wrong and will never actually work. Adding checks & validation for semantic errors is much more complicated and would require dozens of small functions for each possible case. Not something I'm willing to spend time on just for this one user input. My take is that users should make some effort to learn how to set a semantically valid cron schedule and take responsibility for making such errors.